spec/models/conference_preference_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'spec_helper'
+
+describe ConferencePreference do
+  it { is_expected.to belong_to(:team) }
+  it { is_expected.to belong_to(:first_conference) }
+  it { is_expected.to belong_to(:second_conference) }
+
+  describe 'validates terms of ticket and terms of travel' do
+    let(:preference_without_terms) { build(:conference_preference) }
+    let(:preference_with_terms) { build(:conference_preference, :with_terms_checked)}
+
+    context 'when both terms are not checked' do
+      it 'do not save conference preferences' do
+        expect(preference_without_terms).not_to be_valid
```

Comment:
Checks for overall object validity may fail because a different error is causing the object to be in an invalid state. Personally, I prefer something in the form of this:

```ruby
expect(subject.errors[:my_attribute_name]).not_to be_empty
```

(You may need a call to `subject#valid?` first to trigger the validations)